TestUtilities/Package.swift

```diff
@@ -11,14 +11,18 @@ let package = Package(
     products: [
         .library(
             name: "TestUtilities",
-            targets: ["TestUtilities"]),
+            targets: ["TestUtilities"]
+        ),
     ],
     dependencies: [
         .package(name: "Datadog", path: ".."),
     ],
     targets: [
         .target(
             name: "TestUtilities",
-            dependencies: ["Datadog"])
+            dependencies: ["Datadog"],
+            path: ".",
```

Comment:
I think it should be default path. Is there a reason to have it or it's just a leftover? 🤔